src/commons/application/actions/VscodeActions.ts

```diff
@@ -0,0 +1,10 @@
+import { createActions } from 'src/commons/redux/utils';
+
+const VscodeActions = createActions('vscode', {
+  setVscode: () => ({})
```

Comment:
Nit: falsy values mean empty payload by default

```suggestion
  setVscode: 0
```